Make EchoStation more resilient against fire

VmpOS2NHSkhkejA9 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This PR is a collection of mapping tweaks to echostation aimed at making it less vulnerable to fire:
- Most notably, the hole above the holodeck is now surrounded by firelocks, which will drop quickly enough to keep a burn test in the holodeck. The AI "satellite" antechamber has been extended forwards to cover the top of this firelock ring.
- Firelocks have been added to all 6 holodeck doors
- The railings & open spaces connecting the cafeteria to the central hallway have been replaced with fulltile windows.
- The staircase connecting decks 1 and 2 now has a set of firelocks at the top. Fires on the 2nd and 3rd deck halls will not reach the surface entrance or main brig entrance unless these are sabotaged.
- Hydroponics and cargo front desks are now fully protected by firelocks. The doors already had firelocks, but the desks and windoors didn't for some reason, which meant fires would easily break in without any help.
- Hydroponics has a roof, making it harder for fires to get in. Hot enough fires will still simply melt through those tiles and break in.
- A particular maintenance hole between z-levels is now removed.
## Why It's Good For The Game

Echostation is cripplingly vulnerable to fire. Not only is it a small station, which are inherently more vulnerable to fire, it has very few firelocks and massive portions of the station's atmos are connected without any protection. Because of this, any fire or atmospheric problem almost guarantees a shuttle call. This PR doesn't fully fix that, but it patches up the most critical weakpoints and should make fire a less overwhelming threat.

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

<details>
<summary>Before</summary>

![Screenshot 2025-06-12 194208](https://github.com/user-attachments/assets/bba5da62-b369-48e4-b296-d70750a7791b)
![image](https://github.com/user-attachments/assets/47cdc365-0787-4e4d-8ecb-60b1e7a6f42c)
![image](https://github.com/user-attachments/assets/843f8346-b6f7-46b8-91ae-0da1af3e2b35)
![image](https://github.com/user-attachments/assets/43f61d9e-68fb-4727-89f6-e82d82a8bfeb)
![image](https://github.com/user-attachments/assets/35990df4-e3b9-4347-8de4-8cdd3f26ff83)


</details>

<details>
<summary>After</summary>

![Screenshot 2025-06-12 190408](https://github.com/user-attachments/assets/e9bea9d6-db02-443c-b83b-3bce5ecb9156)
![Screenshot 2025-06-12 190352](https://github.com/user-attachments/assets/26693066-630f-4612-9c2b-aa5493290265)
![Screenshot 2025-06-12 190435](https://github.com/user-attachments/assets/e4a0e123-ea01-4661-88f3-9bed924658a9)
![Screenshot 2025-06-12 190456](https://github.com/user-attachments/assets/2eb5abfb-d42e-489a-808f-b9ecfa832969)

</details>

<details>
<summary>In-game</summary>

![image](https://github.com/user-attachments/assets/a8b45457-27fb-486a-a747-da79f2df1f0f)
![image](https://github.com/user-attachments/assets/cfede6c5-2a49-4429-8997-64222438468f)
![image](https://github.com/user-attachments/assets/ab66bf85-b8c5-45f7-b5b3-5ebce707a6fb)
![image](https://github.com/user-attachments/assets/5c1b1e04-bb24-415b-a382-bd2a4ef0eac5)
![image](https://github.com/user-attachments/assets/f602ba6d-36ab-49c5-a4c1-b5c1413aa831)
![image](https://github.com/user-attachments/assets/9304d406-0e47-43e8-87dc-a32834b4f8ee)
![image](https://github.com/user-attachments/assets/fda63ff5-8588-494a-825f-32cb782a1486)



</details>
</details>

## Changelog
:cl:
balance: echostation is now more resilient against fires; the holodeck is now surrounded by firelocks, more departmental front desks have firelocks and a few areas are now broken up by windows and firelocks.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Commit:
Update EchoStation.dmm

Commit:
Update EchoStation.dmm

Commit:
how did that get there???

Commit:
fix linters (probably)

they should invent linters that dont fail so that i can break the game and ruin everything with bugs and errors

BriggsIDP reviewed (approved):
Add in game testing evidence. Lints failing only due to ddos.

Edit: just spotted a floating intercom and a missing fire alarm button on the other side of the firelocks at the main entrance.

Commit:
Update EchoStation.dmm

Commit:
give botany & the kitchen ceilings

botany still only basic plating, so it will melt through fairly quickly, but it will give anyone in the room time to scamper into maintenance

Commit:
i hate linters

because i hate the truth, and the truth is that i made a mistake

Commit:
its STILL not passing linters????
